### PR TITLE
add CSters event mentorship cake night

### PR DESCRIPTION
**Pull request by mokaixu**

### CODE REVIEW

**Comment by arshiamufti** [comment]
I like the description that Shivangi gave in the group chat "help underclass women in CS learn more about various options, specific upper year classes, interviews and other aspects of university life". I find it a little more detailed. Maybe you could add that in!

**Comment by evykassirer** [comment]
u g h the dropbox event has a date that's in february oops :(

can you change the date on that file to be jan 24 instead of feb 24, so that this post is more recent?

**Comment by mokaixu** [comment]
sure!

File: content/W2017/W17-events-mentorship-cake.md [hunk 1]
```diff
@@ -4,7 +4,8 @@ Category: Events
 Tags: social, big csters, 
 Slug: mentoship-night-w17
 Author: Monica Xu
-Summary: Meet other women in CS, connect with upper year CS students, and enjoy some cake!
+Summary: Get upperclassmen help about various CS options, specific upper year classes,
+interviews and other aspects of university life
```
Comment (annalorimer) [inline content/W2017/W17-events-mentorship-cake.md]: For the website to render properly, this needs to be one line. Or else you get this: 
screen shot 2017-02-03 at 12 51 06 pm

**Comment by evykassirer** [comment]
looks good - can you just change the dropbox date to be last month instead?

**Pull request merged**